src/global.css

```diff
@@ -24,6 +34,15 @@ input[type="radio"] + label{
   margin-left: 1rem;
 }
 
+input[type="submit"]{
+  padding: 0.5rem;
+  text-transform: uppercase;
+}
+
 dd{
   -webkit-margin-start: 0;
+}
+
+[data-reactroot]{
```

Comment:
It would be cleaner to create a CSS class and apply it to the root component. It makes your CSS classes more portable and easier to refactor.